.claude/commands/pr-workflow.md

```diff
@@ -438,6 +463,8 @@ If any validation fails, **STOP** and provide clear instructions to fix the issu
    **Example commit for Option 1:**
    ```bash
    git commit -m "$(cat <<EOF
+   ```
```

Comment:
this looks wrong! it's in the middle of an EOF

Reply:
Fixed in commit 85cc938: corrected bash heredoc syntax that was broken by dprint formatting.

The heredoc examples were split incorrectly across code fence boundaries. I've fixed both instances in the file.